web/html/src/components/input/Form.tsx

```diff
@@ -137,12 +137,14 @@ export class Form extends React.Component<Props> {
             }
             return filtered;
           }, {});
-          const errors = Object.keys(this.props.errors || {}).reduce((filtered, key) => {
-            if (names.includes(key) && this.props.errors?.[key]) {
-              return filtered.concat(this.props.errors[key]);
-            }
-            return filtered;
-          }, []);
+          const errors = this.props.errors != null ?
+            Object.keys(this.props.errors || {}).reduce((filtered, key) => {
+              if (names.includes(key) && this.props.errors?.[key]) {
+                return filtered.concat(this.props.errors[key]);
+              }
+              return filtered;
+            }, []) :
+            undefined;
```

Comment:
It's not clear what the intent here is. What's the benefit of having `undefined` instead of an empty array `[]` annotating no errors?

Reply:
I'll use `[]` it's fine.